web-registry/src/pages/Media.tsx

```diff
@@ -38,8 +41,20 @@ const Media: React.FC = () => {
     return Promise.resolve();
   };
 
+  function navigateNext(): void {
+    // TODO: replace 'review' with path name once
+    // https://github.com/regen-network/regen-registry/issues/899 is merged
+    const nextStep = creditClassId ? 'metadata' : 'review';
```

Comment:
there's another version of the same /metadata page for C01 class id: https://github.com/regen-network/regen-registry/issues/908

Reply:
Ok so we can always navigate to this page at this step and we will have to update the metadata page once we add the second form then to show the good one.

Reply:
as part of the legacy project creation flow, there is no metadata form (in case of no on chain credit class id) which is why we navigate to /review in this case
for the new on chain project creation flow, the metadata page will need to be updated indeed to show the VCS one if needed